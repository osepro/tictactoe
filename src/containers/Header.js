import React, { useState } from "react";

const Header = ({ game }) => {
	return (
		<div>
			<h1>TIC TAC TOE</h1>
			<p>{game}</p>
		</div>
	);
};

export default Header;
