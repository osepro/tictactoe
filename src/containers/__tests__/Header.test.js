import React from "react";
import { shallow } from "enzyme";
import ReactDOM from "react-dom";
import Header from "containers/Header";

let wrapped;

beforeEach(() => {
	wrapped = shallow(<Header />);
});

it("shows TIC TAC TOE", () => {
	expect(wrapped).toMatchSnapshot();
});

it("containers header title", () => {
	expect(wrapped.render().text()).toContain("TIC TAC TOE");
});
