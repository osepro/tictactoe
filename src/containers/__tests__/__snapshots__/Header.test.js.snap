// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shows TIC TAC TOE 1`] = `ShallowWrapper {}`;
